yarn.lock

```diff
@@ -3264,7 +3264,7 @@ jest-cli@^24.7.1:
     realpath-native "^1.1.0"
     yargs "^12.0.2"
 
-jest-config@^24.7.0, jest-config@^24.7.1:
+jest-config@^24.7.1:
```

Comment:
Should this be totally removed? 🤔

Reply:
The remaining one is a dependency of `jest`. But this time it is only a devDependency.